File: problemas/iniciante/2763/2763.js
process.stdin.resume();
process.stdin.setEncoding('utf8');

let input = "";

process.stdin.on('data', (chunk) => {
    input += chunk;
});

process.stdin.on('end', () => {
    let entrada = input.split("\n");

    let CPF = entrada.shift().trim().split(/[\\.-]/);

    for (let valor of CPF) {
        console.log(valor);
    }
});

File: problemas/iniciante/2763/2763.py
def main():
    saida = ''
    resultado = []
    CPF = str(input().replace('.', ''))
    for elemento in CPF:
        saida += elemento
        if len(saida) == 3:
            resultado.append(saida)
            saida = ''
    
    for valor in resultado:
        print(valor.replace('-', ''))
        
if __name__ == "__main__":
    main()
